.github/workflows/ci.yml

```
@@ -54,7 +54,11 @@ jobs:
           SofaPython3_ROOT="$GITHUB_WORKSPACE/SofaPython3"
           mkdir -p "${{ runner.temp }}/sp3_tmp/zip" "${{ runner.temp }}/sp3_tmp/binaries" "$SofaPython3_ROOT"
           url="https://github.com/sofa-framework/SofaPython3/releases/download"
-          url="${url}/release-master-nightly/SofaPython3_master-nightly_python-${{ matrix.python_version }}_for-SOFA-${{ matrix.sofa_branch }}_${{ runner.os }}.zip"
+          if [[ "${{ matrix.sofa_branch }}" == "master" ]]; then
+            url="${url}/release-master-nightly/SofaPython3_master-nightly_python-${{ matrix.python_version }}_for-SOFA-master_${{ runner.os }}.zip"
+          else
+            url="${url}/release-${{ matrix.sofa_branch }}/SofaPython3_${{ matrix.sofa_branch }}_python-${{ matrix.python_version }}_for-SOFA-${{ matrix.sofa_branch }}_${{ runner.os }}.zip"
+          fi
```

```suggestion
          url="${url}/release-${{ matrix.sofa_branch }}/SofaPython3_${{ matrix.sofa_branch }}_python-${{ matrix.python_version }}_for-SOFA-${{ matrix.sofa_branch }}_${{ runner.os }}.zip"
```
Why searching for `master-nightly` and not `master`? I suspect that `master-nightly` is not even built nightly.

---

That's a good question. Honestly I don't know, I kept the `master-nightly` as it was already there. 
I don't see any drawback switching to non-nightly build, and it would make the code simpler.

---

The term nightly comes from 
https://github.com/sofa-framework/SofaPython3/blob/c5c1603cbfaa1ef5f11a5402186e83635edfba3b/.github/workflows/ci.yml#L6, which is triggered only manually. Unless there is a reason I don't see, I would not base the other plugins on this package.

---

Alright. I've made the changes.